theano/gpuarray/opt.py

```diff
@@ -615,7 +613,7 @@ def local_gpua_alloc(op, context_name, inputs, outputs):
 def local_gpua_alloc_empty(op, context_name, inputs, outputs):
     # We use _props_dict() to make sure that the GPU op know all the
     # CPU op props.
-    return gpu_alloc_empty(context_name, **op._props_dict())
+    return GpuAllocEmpty(**op._props_dict())(*inputs)
```

Comment:
I think it should be:

GpuAllocEmpty(context_name=ontext_name, **op._props_dict())(*inputs)

You can test it locally with: 

nosetests theano/scan_module/tests/test_scan.py:T_Scan_Gpuarray.test_one_sequence_one_output_weights_gpu1

this is one of the jenkins failing test.